cli/cliui/agent.go

```diff
@@ -381,6 +381,9 @@ func (d ConnDiags) Write(w io.Writer) {
 			_, _ = fmt.Fprintf(w, " - %s\n\n", msg)
 		}
 	}
+	if !d.PingP2P || d.Verbose || len(client) > 0 || len(agent) > 0 || len(general) > 1 {
+		_, _ = fmt.Fprintln(w, "For more information, see https://coder.com/docs/networking/troubleshooting")
```

Comment:
I was sort of hoping that we'd have a specific troubleshooting link per detected problem that takes them right to the issue. e.g. https://coder.com/docs/networking/troubleshooting#endpoint_dependent_nat__hard_nat_

Reply:
Good idea, this is a lot nicer